appinfo/signature.json

```diff
@@ -0,0 +1,351 @@
+        "vendor\/symfony\/yaml\/Unescaper.php": "7c13301a22c2a59beca9354be8527bf3079e29f3ad7b2b797d13fc74b1b20398854e37928b2d0734144bae0c17f4e64254bd4091599c3716be01afc4a4155caa",
+        "vendor\/symfony\/yaml\/Yaml.php": "44ba0b87a7da1ca19568e147a2a9ceff6bbbf5e189608e61de7c6e1a48936f18c0535e9470c58e078c93a895f5c3f6a3bbee6c2314c7150933025b1dd8bab36e",
+        "vendor\/symfony\/yaml\/composer.json": "cba998bad7693859f70bdf6bf02a568c058cdfbba9350eac5ee49942f88c1863554e84f950586b9917ca47d714416534a9e5e2900c085f8b50a86a870b8ee070",
+        "vendor\/symfony\/yaml\/phpunit.xml.dist": "37f885ad06632564d808ac1cb39e90277a6c084f1e1de486141b7473c44fb3f715b4bc870a75f80fe4e82c9e42e345e8c896dccff64c472004586bb156644da6"
+    },
+    "signature": "QBBRwFT3\/0m1F4Cc3QLVxFCtKHnilX1pFxp+vtMnE8qGEAxu\/dttQW914eC2T6YK3+PBbwLxjO76CHk0lF08neq2o+HVwIGb2uaBxk2PGZhwVNgyQS3zBlRzMUzIi506fz6XEbV11S8c0\/2CrRWAnsIxzRg5iUDw6kGRqlEF+miLQWvz2TKTGwkVAhpBwliVouw+ZrqileN3ZpvjuNVH5HMkoiD78ofy8wku3DhJES1+jJYmZ2qxA9L9uPJjLAgu8fcU+JoL6WD4Hf1RP0YEhGdjWMcX087Qs0N5C4Ynr3t7UcBltXS9RVdooamWjpIMon\/6hyn6aP2L7+CMg8ShTw==",
+    "certificate": "-----BEGIN CERTIFICATE-----\r\nMIIEojCCA4qgAwIBAgICEAAwDQYJKoZIhvcNAQELBQAwezELMAkGA1UEBhMCREUx\r\nGzAZBgNVBAgMEkJhZGVuLVd1ZXJ0dGVtYmVyZzEXMBUGA1UECgwOTmV4dGNsb3Vk\r\nIEdtYkgxNjA0BgNVBAMMLU5leHRjbG91ZCBDb2RlIFNpZ25pbmcgSW50ZXJtZWRp\r\nYXRlIEF1dGhvcml0eTAeFw0xNjA2MTIyMTA1MDZaFw00MTA2MDYyMTA1MDZaMGYx\r\nCzAJBgNVBAYTAkRFMRswGQYDVQQIDBJCYWRlbi1XdWVydHRlbWJlcmcxEjAQBgNV\r\nBAcMCVN0dXR0Z2FydDEXMBUGA1UECgwOTmV4dGNsb3VkIEdtYkgxDTALBgNVBAMM\r\nBGNvcmUwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDUxcrn2DC892IX\r\n8+dJjZVh9YeHF65n2ha886oeAizOuHBdWBfzqt+GoUYTOjqZF93HZMcwy0P+xyCf\r\nQqak5Ke9dybN06RXUuGP45k9UYBp03qzlUzCDalrkj+Jd30LqcSC1sjRTsfuhc+u\r\nvH1IBuBnf7SMUJUcoEffbmmpAPlEcLHxlUGlGnz0q1e8UFzjbEFj3JucMO4ys35F\r\nqZS4dhvCngQhRW3DaMlQLXEUL9k3kFV+BzlkPzVZEtSmk4HJujFCnZj1vMcjQBg\/\r\nBqq1HCmUB6tulnGcxUzt\/Z\/oSIgnuGyENeke077W3EyryINL7EIyD4Xp7sxLizTM\r\nFCFCjjH1AgMBAAGjggFDMIIBPzAJBgNVHRMEAjAAMBEGCWCGSAGG+EIBAQQEAwIG\r\nQDAzBglghkgBhvhCAQ0EJhYkT3BlblNTTCBHZW5lcmF0ZWQgU2VydmVyIENlcnRp\r\nZmljYXRlMB0GA1UdDgQWBBQwc1H9AL8pRlW2e5SLCfPPqtqc0DCBpQYDVR0jBIGd\r\nMIGagBRt6m6qqTcsPIktFz79Ru7DnnjtdKF+pHwwejELMAkGA1UEBhMCREUxGzAZ\r\nBgNVBAgMEkJhZGVuLVd1ZXJ0dGVtYmVyZzESMBAGA1UEBwwJU3R1dHRnYXJ0MRcw\r\nFQYDVQQKDA5OZXh0Y2xvdWQgR21iSDEhMB8GA1UEAwwYTmV4dGNsb3VkIFJvb3Qg\r\nQXV0aG9yaXR5ggIQADAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUH\r\nAwEwDQYJKoZIhvcNAQELBQADggEBADZ6+HV\/+0NEH3nahTBFxO6nKyR\/VWigACH0\r\nnaV0ecTcoQwDjKDNNFr+4S1WlHdwITlnNabC7v9rZ\/6QvbkrOTuO9fOR6azp1EwW\r\n2pixWqj0Sb9\/dSIVRpSq+jpBE6JAiX44dSR7zoBxRB8DgVO2Afy0s80xEpr5JAzb\r\nNYuPS7M5UHdAv2dr16fDcDIvn+vk92KpNh1NTeZFjBbRVQ9DXrgkRGW34TK8uSLI\r\nYG6jnfJ6eJgTaO431ywWPXNg1mUMaT\/+QBOgB299QVCKQU+lcZWptQt+RdsJUm46\r\nNY\/nARy4Oi4uOe88SuWITj9KhrFmEvrUlgM8FvoXA1ldrR7KiEg=\r\n-----END CERTIFICATE-----"
+}
```

Comment:
As this changes once a file is touched it should not be checked into git but only be generated during packaging the app for the app store release.

Reply:
Thanks for the info, the file has been removed from repository